### PR TITLE
doc: Update dependency graph.

### PR DESCRIPTION
**Pull request by yifan-gu**
Add terraform tfvars file as a dependency of the tfstate.

### CODE REVIEW

File: Documentation/design/resource_dep.dot [hunk 1]
```diff
@@ -196,11 +204,10 @@ strict digraph resource {
...
   kubeconfig_admin -> manifests;
-  kubeconfig_admin -> cluster;
 
   bootstrap_ignition -> ignition_configs;
   master_ignition -> ignition_configs;
   worker_ignition -> ignition_configs;
 
-  terraform_state -> cluster;  
+  openshift_cluster -> cluster;
 }
```
Comment (crawford) [inline Documentation/design/resource_dep.dot]: I think we should reserve the cluster target for if/when the installer invokes terraform on behalf of the user. Maybe call this target cluster-inputs?

File: Documentation/design/resource_dep.dot [hunk 2]
```diff
@@ -79,11 +79,16 @@ strict digraph resource {
...
   {
     node [style=filled,color=gold];
-    terraform_state [label="*.tfstate"];
+    terraform_tfvars [label="terraform.tfvars"];
+  }
+
+  // OpenShift cluster.
+  {
+    node [style=filled, color=lemonchiffon];
+    openshift_cluster [label="cluster"];
```
Comment (wking) [inline Documentation/design/resource_dep.dot]: The distinction between `cluster` and `openshift_cluster` is not clear to me, especially with `openshift_cluster` -> cluster`.  Can we get more descriptive names for these?  Or maybe drop `cluster` and rename `openshift_cluster` -> `cluster`?

**Comment by crawford** [comment]
@yifan-gu Can you update this or close it?

**Comment by yifan-gu** [comment]
Going to drop this, I feel the current is good enough.

**Pull request closed without merging**